Changes.md

```diff
@@ -1,3 +1,14 @@
+# v0.12.0
+
+Features:
+- Add compression of pre-compressed frame data, via `Encoder::write_lzw_pre_encoded_frame`.
+- The `color_quant` dependency is now optional. Turning it off disables some
+  interfaces that would internally build quantization tables, which can be
+  costly compared in specialized situations.
```

Comment:
I'm having trouble parsing this sentence. Did you mean "which can be costly ~compared~ in specialized situations"

Reply:
Agreed, I've rewritten that section.